.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "python.formatting.provider": "black",
+    "python.formatting.provider": "none",
```

Comment:
why do we set it here as "none" and later set it again as "black"? is it a matter of the version of VSCode?

Reply:
As per [vscode-python](https://aka.ms/AAlgvkb), settings like this will soon be deprecated in favor of extensions. I will revert it in case someone is not up-to-date, I don't imagine it can cause any harm.

Reply:
cool. let's merge then!